shared/uavobjectdefinition/actuatorsettings.xml

```diff
@@ -23,8 +23,8 @@
             <description>When enabled, the motors will spin at the ChannelNeutral command when armed with zero throttle.</description>
         </field>
 
-        <field name="MotorInputOutputCurveFit" units="-" type="float" elementnames="A,B" defaultvalue="1,1">
-            <description>Actuator mapping of input in [-1,1] to output on [-1,1], using power equation of type a*x^b.</description>
+        <field name="MotorInputOutputCurveFit" units="-" type="float" elements="1" defaultvalue="1">
```

Comment:
We're free to discuss it here.  Multiplication commutes with two levels of control terms and introduces weird clipping behavior (that the previous levels don't).  Multiplication by a constant can't, by definition, improve linearity.

I know I've asked you about this before but we simply can't come up with a justification for this.